src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health, name) {
```

Comment:
[CONSTRUCTOR PARAMETERS] - The constructor of the `Animal` class should not accept a `health` parameter since health is always initialized to `100`. Please remove the `health` parameter from the constructor signature.